src/server/scripts/Spells/spell_monk.cpp

```diff
@@ -159,9 +159,62 @@ class spell_monk_provoke : public SpellScript
     }
 };
 
+// Roll - 109132
+class spell_monk_roll : public SpellScriptLoader
+{
+public:
+    spell_monk_roll() : SpellScriptLoader("spell_monk_roll") { }
+
+    class spell_monk_roll_SpellScript : public SpellScript
+    {
+        PrepareSpellScript(spell_monk_roll_SpellScript);
+
+        bool Validate(SpellInfo const* /*spell*/) override
+        {
+            if (!sSpellMgr->GetSpellInfo(109132, DIFFICULTY_NONE))
+                return false;
```

Comment:
Since you have a single condition and return boolean, preferably use a single statement and return the result of the condition (in this case GetSpellInfo or ValidateSpellInfo)